src/Uno.UI.Wasm/ts/Clipboard.ts

```diff
@@ -17,5 +49,28 @@
 
 			return "ok";
 		}
+
+		public static getText(requestId: string): void {
+			if (!Clipboard.dispatchGetContent) {
+				Clipboard.dispatchGetContent = (<any>Module).mono_bind_static_method("[Uno] Windows.ApplicationModel.DataTransfer.Clipboard:DispatchGetContent");
+			}
+
+			var nav = navigator as NavigatorClipboard;
+			if (nav.clipboard) {
+				var promise = nav.clipboard.readText();
+				promise.then(
+					(clipText : string) => Clipboard.dispatchGetContent(requestId, clipText),
+					(_ : any) => Clipboard.dispatchGetContent(requestId, null));
+			} else {
+				Clipboard.dispatchGetContent(requestId, null);
+			}
+		}
+
+		private static onClipboardChanged() {
+			if (!Clipboard.dispatchContentChanged) {
+				Clipboard.dispatchContentChanged = (<any>Module).mono_bind_static_method("[Uno] Windows.ApplicationModel.DataTransfer.Clipboard:DispatchContentChanged");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5540286)